### ISSUE TITLE
can you help me to find what's wrong with my code

### ISSUE THREAD

**Issue by xuxunima** [wxinbeings/mobilenetv3_keras#1]
I implement the mobilenetv3 in keras, but when I train the model, the loss and accuracy keep the same,I'm new in deep learning, I can't find the bug.Thank you!

**Comment by wxinbeings** [comment]
Without your code, it's hard to tell.
